src/views/Project/Details/Frameworks/FrameworkDetail/index.js

```diff
@@ -127,6 +130,59 @@ const projectRendererParams = (_, p) => ({
 
 const hideQuestionnaire = ({ accessQuestionnaire }) => !accessQuestionnaire;
 
+function saveAs(uri, filename) {
+    const link = document.createElement('a');
+    if (typeof link.download === 'string') {
+        link.href = uri;
+        link.download = filename;
+
+        document.body.appendChild(link);
+
+        link.click();
+
+        document.body.removeChild(link);
+    } else {
+        window.open(uri);
+    }
+}
+
+const getFrameworkSize = (framework) => {
+    let width = 0;
+    let listHeight = 0;
+    let overviewHeight = 0;
+
+    framework.widgets.forEach((w) => {
```

Comment:
Prefer a `Math.max(...framework.widgets.map(function to get the extremes));` apporach